src/app/core/services/electron.service.ts

```diff
@@ -76,13 +76,6 @@ export class ElectronService {
     }
   }
 
-  getCurrentDirectory(): string {
-    if (this.isElectron()) {
-      this.ipcRenderer.sendSync('synchronous-message', 'getDirectory');
```

Comment:
Let's also remove the main Electron process code used to retrieve the current directory